view/script/patient.js

```diff
@@ -38,7 +39,9 @@
         }
     };
 
+    var allDatesConfig = jQuery.extend(true, {}, window.dates);
```

Comment:
okay, I have been alerted that this is being used as a deep copy, could a comment be added to that affect?

thanks! :smile: